src/app/component/dynamic-elements/dynamic-grid/dynamic-grid.component.ts

```diff
@@ -155,7 +163,11 @@ export class DynamicGridComponent implements OnInit {
     return parameters;
   }
 
-  actionBegin(args: SaveEventArgs): void {
+  actionBegin(args: any): void {
+    if(args.currentPage) {
+      this.savePage['home/reservations'] = args.currentPage;
```

Comment:
Also applies to this.
So we can't use a hardcoded solution in this case, we need to take the part we need from the URL.